umu-database.csv

```diff
@@ -1002,6 +1002,7 @@ Flowers - Le Volume Sur Hiver,gog,1564851593,umu-1564851593,,
 Dragon Age: Inquisition,ea,1000659,umu-1222690,,
 Through the Darkest of Times,amazon,amzn1.adg.product.d306715a-5b83-467f-9795-ffb3e0b9575b,umu-1003090,,
 Bus Simulator 21: Next Stop,amazon,amzn1.adg.product.730d8afc-aa42-45e2-815b-a808dbbe4c76,umu-976590,,
+Bus Simulator 21: Next Stop,egs,10adac482d1647e1afc237b66dc45066,umu-976590,,
```

Comment:
I believe the ID should be `0fcb6881984f4e9ab91f5e22e4bf844c`.

See https://egdata.app/items/5fee571eae9d43e798ea022d632232fa/assets